chalicelib/checks/audit_checks.py

```diff
@@ -336,15 +336,26 @@ def check_help_page_urls(connection, **kwargs):
     check = CheckResult(connection, 'check_help_page_urls')
 
     server = connection.ff_keys['server']
-    results = ff_utils.search_metadata('search/?type=StaticSection&q=help&status!=draft&field=body',
+    results = ff_utils.search_metadata('search/?type=StaticSection&q=help&status!=draft&field=body&field=options',
                                        key=connection.ff_keys)
     sections_w_broken_links = {}
     for result in results:
         broken_links = []
-        urls = re.findall('[\(|\[|=]["]*(http[^\s\)\]]+|/[^\s\)\]]+)[\)|\]|"]', result.get('body', ''))
+        body = result.get('body', '')
+        urls = []
+        if result.get('options', {}).get('filetype') == 'md':
+            links = re.findall('\[[^\]]+\]\([^\)]+\)', body)
```

Comment:
I'd just add some comments here about searching the markdown for the [text](link) pattern.